microsoft/atlas-design
# Add segmented control component

Task: task-[716984](https://dev.azure.com/ceapex/Engineering/_sprints/taskboard/Assessments/Engineering/2023-01-11?workitem=716984)

Link: local [494](http://localhost:1111/components/segmented-controls.html)

[Figma](https://www.figma.com/proto/uVA2amRR71yJZ0GS6RI6zL/%F0%9F%8C%9E-Atlas-Design-Library?page-id=1635%3A31717&node-id=24586%3A195606&viewport=-4121%2C-1272%2C0.47&scaling=scale-down-width&starting-point-node-id=24586%3A195606)

Add segmented control component styles and functionality.

Additionally, incorporates GitHub's `tab-container` into the Atlas code, with some additional code to allow for a next and previous button. tab-container is not strictly coupled to the styling of the control, so it will be a re-usable base for other tab components.

## Testing
1. Pull this branch, `npm i` and then run `npm run start`
1. Click the link above for local page. [494](http://localhost:1111/components/segmented-controls.html)
1. Click navigation arrows
  Expected result: the active item should change
1. Click tab
   Expected result: acive item should change

## Additional information

[Optional]
